DI-Policies.md

```diff
@@ -0,0 +1,67 @@
+# Statement
+
+Historically, the wedding industry has been massively lacking in areas of representation and equity, especially across race, gender, and sexual orientation. The fact that interracial marriage wasn’t fully legal in the U.S. until 1967, and same sex marriage wasn’t legal until 2015, has been reflected in wedding industry practices and imagery since its beginning. And to this day, it’s painfully clear just how racist, sexist, homophobic, and generally prejudiced the industry still is.
+
+It’s up to wedding vendors and industry leaders to correct this. We have a responsibility to create spaces and content where all people feel seen, safe, and valued. 
+
+It’s important to me as a wedding business owner to not only implement policies that address these issues, but also hold myself accountable to those policies. So, I’m sharing this document publicly as one part of my own efforts to do better and be better. This not an “arrival”, but rather part of an ongoing process that will evolve and change over the lifetime of my business as I learn and iterate to create more inclusive, welcoming, and safe spaces for all folx.
+
+# Specific policies
+
+These are the specific actions I commit to taking as integral parts of how I do business moving forward. 
+
++ Listening + amplifying
+
+   + Listen to and amplify the stories and experiences of marginalized groups in the industry, and out of it.
+      + Examples include, but are not limited to:
+         + Promotion of diversity-and-inclusion-minded vendors in my social feeds and blog
+         + Sharing love stories from underrepresented folx in my social feeds and blog
+
+   + Use my privilege, and speak up when witness to problematic statements, situations, and practices.
+
+   + Own my own mistakes when - not if - I make them, make reparations, and do better next time.
+
++ Regular audits of website and social media feeds
+
+   + Quarterly for website:
+      + Ensure representation of underrepresented groups in imagery across website, including blog
+      + Ensure the use of inclusive language (e.g. gender neutral terms) across website, including blog
+
+   + Monthly for social:
+      + Ensure representation of underrepresented groups in feeds, across all platforms (Instagram, Facebook, Twitter, Pinterest)
+      + Ensure the use of inclusive language extends to all social posts
+
++ Vetting subcontractors + partners
+
+   + Interview second shooters, editors, and any others I work with or hire to ensure inclusive practices and the comfort/safety of my clients.
+      + Questions partners will be expected to answer include:
```

Comment:
@Pseudopam @bryanaknight What do you think of these questions? Any you would add? Anything you'd change/remove?

Reply:
These are excellent! 

I wonder if 5 is _too_ many? Just thinking about holding yourself accountable, and also starting somewhere and iterating, do you think you'll be comfortable adding all these questions into the interview process for potential partners? I just want to make sure you're setting yourself up for success 😄 

If you did want to shorten this list, some suggestions:
- Remove Q1 (I feel like you could get a sense for this based on the answers of the others)
- Consolidate Q3 and Q4 (or just remove Q4 as Q3 will give you a sense)

Again, I think these are ALL great questions ✨

Reply:
@bryanaknight Really good point. I think you're right. I don't want to be redundant, nor do I want it to feel overwhelming to folks. I like your suggestions for consolidation! 

Before I make any changes, @Pseudopam I'd love your thoughts, too.

Reply:
@Pseudopam I'm actually going ahead and making some changes here, but I'd still love your input on this and any other changes I'm making at this time.